scenarios/evaluate-safety/evaluate-protected-material-and-indirect-attack-jailbreak.ipynb

```diff
@@ -0,0 +1,512 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate Protected Material and Indirect Attack Jailbreak\n",
+    "\n",
+    "## Objective\n",
+    "This notebook walks through how to generate a simulated conversation targeting a deployed AzureOpenAI model and then evaluate that conversation dataset for Protected Material and Indirect Attack Jailbreak vulnerability. It also references the prompt filtering capabilities of Azure AI to help identify and mitigate these vulnerabilities in your AI system.\n",
+    "\n",
+    "## Time\n",
+    "You should expect to spend about 30 minutes running this notebook. If you increase or decrease the number of simulated conversations, the time will vary accordingly.\n",
+    "\n",
+    "## Before you begin\n",
+    "\n",
+    "### Installation\n",
+    "Install the following packages required to execute this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install the packages\n",
+    "%pip install openai promptflow-evals azure-identity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set the following environment variables for use in this notebook:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"AZURE_DEPLOYMENT_NAME\"] = \"\"\n",
+    "os.environ[\"AZURE_ENDPOINT\"] = \"\"\n",
+    "os.environ[\"AZURE_API_VERSION\"] = \"\"\n",
+    "os.environ[\"AZURE_API_KEY\"] = \"\"\n",
+    "os.environ[\"AZURE_SUBSCRIPTION_ID\"] = \"\"\n",
+    "os.environ[\"AZURE_RESOURCE_GROUP\"] = \"\"\n",
+    "os.environ[\"AZURE_PROJECT_NAME\"] = \"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuration\n",
+    "The following simulator and evaluators require an Azure AI Studio project configuration and an Azure credential to use. \n",
+    "Your project scope will be what is used to log your evaluation results in your project after the evaluation run is finished."
```

Comment:
Add: "For this sample, we recommend creating or using a project in East US 2. For full region supportability, see our [documentation](https://aka.ms/azureaistudiosafetyevalhowto)."